aws/logs_monitoring/lambda_function.py

```diff
@@ -266,8 +271,8 @@ class DatadogHTTPClient(object):
     _POST = "POST"
     _HEADERS = {"Content-type": "application/json"}
 
-    def __init__(self, host, api_key, scrubber, timeout=10):
-        self._url = "https://{}/v1/input/{}".format(host, api_key)
+    def __init__(self, host, port, api_key, scrubber, timeout=10):
+        self._url = "https://{}:{}/v1/input/{}".format(host, port, api_key)
```

Comment:
Do we want to send traffic in clear over `http` ? If so, I think we probably want to use `http` as scheme instead of `https`

Reply:
Updated